windows/en-US/firefox.adml

```diff
@@ -100,6 +100,7 @@
       <string id="ProxySettings_group">Proxy Settings</string>
       <string id="SecurityDevices_group">Security Devices</string>
       <string id="FirefoxSuggest_group">Firefox Suggest (US only)</string>
+      <string id="ContentAnalysis_group">Content Analysis</string>
```

Comment:
I might use `Content Analysis (DLP)` here - the [SUMO page](https://support.mozilla.org/en-US/kb/data-loss-prevention) refers to it as DLP ("Content Analysis" is just our internal name for it)